src/components/TileGrid/UserTile.tsx

```diff
@@ -0,0 +1,66 @@
+import { Senior, User } from "@prisma/client";
+import Image from "next/legacy/image";
+import Link from "next/link";
+import { useSession } from "next-auth/react";
+import React, { ReactNode } from "react";
+import "@fontsource/merriweather";
+
+interface UserTileProps {
+  student?: User;
+  senior?: Senior;
+  link: string;
+  dropdownComponent?: ReactNode;
+}
+
+export function UserTile({
+  student,
+  senior,
+  link,
+  dropdownComponent,
+}: UserTileProps) {
+  const { data } = useSession();
```

Comment:
nit: I don't think we need `useSession()` here. It seems like the purpose is to check for authentication - but pages that render `UserTile` is ensured that it's protected.